kitsune/products/views.py

```diff
@@ -85,19 +85,21 @@ def document_listing(request, topic_slug, product_slug=None, subtopic_slug=None)
 
     if topic_slug:
         try:
-            old_topic_slug = TopicSlugHistory.objects.get(slug=topic_slug)
+            old_topic_slug = TopicSlugHistory.objects.get(
+                Q(slug=topic_slug) | Q(slug=subtopic_slug)
+            )
             redirect_params = {
                 "topic_slug": old_topic_slug.topic.slug,
             }
 
             if product_slug:
                 redirect_params["product_slug"] = product_slug
-            if subtopic_slug:
-                redirect_params["subtopic_slug"] = subtopic_slug
 
             return redirect(document_listing, **redirect_params)
         except TopicSlugHistory.DoesNotExist:
             ...
+        except TopicSlugHistory.MultipleObjectsReturned:
+            ...
```

Comment:
Same as above - legit?

Reply:
same `...` is the same with `pass` in Python

Reply:
I know - just making sure you didn't intend to do something with those exceptions.